pimd/pim_cmd.c

```diff
@@ -1608,13 +1611,9 @@ static void pim_show_neighbors_secondary(struct vty *vty)
 		     neigh_src_str, sizeof(neigh_src_str));
 
       for (ALL_LIST_ELEMENTS_RO(neigh->prefix_list, prefix_node, p)) {
-	char neigh_sec_str[INET_ADDRSTRLEN];
+	char neigh_sec_str[100];
 
-	if (p->family != AF_INET)
-	  continue;
-
-	pim_inet4_dump("<src?>", p->u.prefix4,
-		       neigh_sec_str, sizeof(neigh_sec_str));
+	prefix2str(p, neigh_sec_str, 100);
```

Comment:
urgh, the "100" sticks out like a sore thumb ... `PREFIX2STR_BUFFER` above and `sizeof` here?